src/tsconfig.strictNullChecks.json

```diff
@@ -6,734 +6,746 @@
-		"./vs/workbench/services/issue/common/issue.ts",
-		"./vs/workbench/services/jsonschemas/common/jsonValidationExtensionPoint.ts",
-		"./vs/workbench/services/keybinding/common/keybindingIO.ts",
-		"./vs/workbench/services/keybinding/common/keyboardMapper.ts",
-		"./vs/workbench/services/keybinding/common/macLinuxFallbackKeyboardMapper.ts",
-		"./vs/workbench/services/keybinding/common/macLinuxKeyboardMapper.ts",
-		"./vs/workbench/services/keybinding/common/windowsKeyboardMapper.ts",
-		"./vs/workbench/services/keybinding/electron-browser/keybindingService.ts",
-		"./vs/workbench/services/keybinding/test/keyboardMapperTestUtils.ts",
-		"./vs/workbench/services/mode/common/workbenchModeService.ts",
-		"./vs/workbench/services/notification/common/notificationService.ts",
-		"./vs/workbench/services/panel/common/panelService.ts",
-		"./vs/workbench/services/part/common/partService.ts",
-		"./vs/workbench/services/remote/electron-browser/remoteAgentServiceImpl.ts",
-		"./vs/workbench/services/remote/node/remoteAgentEnvironmentChannel.ts",
-		"./vs/workbench/services/remote/node/remoteAgentService.ts",
-		"./vs/workbench/services/scm/common/scm.ts",
-		"./vs/workbench/services/scm/common/scmService.ts",
-		"./vs/workbench/services/search/common/searchHelpers.ts",
-		"./vs/workbench/services/search/node/fileSearchManager.ts",
-		"./vs/workbench/services/search/node/legacy/search.ts",
-		"./vs/workbench/services/search/node/ripgrepFileSearch.ts",
-		"./vs/workbench/services/search/node/ripgrepSearchProvider.ts",
-		"./vs/workbench/services/search/node/ripgrepSearchUtils.ts",
-		"./vs/workbench/services/search/node/ripgrepTextSearchEngine.ts",
-		"./vs/workbench/services/search/node/search.ts",
-		"./vs/workbench/services/search/node/searchHistoryService.ts",
-		"./vs/workbench/services/search/node/searchIpc.ts",
-		"./vs/workbench/services/search/node/textSearchAdapter.ts",
-		"./vs/workbench/services/search/node/textSearchManager.ts",
-		"./vs/workbench/services/search/test/node/ripgrepTextSearchEngine.test.ts",
-		"./vs/workbench/services/search/test/node/textSearchManager.test.ts",
-		"./vs/workbench/services/textMate/electron-browser/TMGrammars.ts",
-		"./vs/workbench/services/textMate/electron-browser/TMHelper.ts",
-		"./vs/workbench/services/textMate/electron-browser/TMSyntax.ts",
-		"./vs/workbench/services/textMate/electron-browser/textMateService.ts",
-		"./vs/workbench/services/textfile/electron-browser/textResourcePropertiesService.ts",
-		"./vs/workbench/services/themes/common/colorExtensionPoint.ts",
-		"./vs/workbench/services/themes/common/colorThemeSchema.ts",
-		"./vs/workbench/services/themes/common/fileIconThemeSchema.ts",
-		"./vs/workbench/services/themes/common/workbenchThemeService.ts",
-		"./vs/workbench/services/title/common/titleService.ts",
-		"./vs/workbench/services/workspace/common/workspaceEditing.ts",
-		"./vs/workbench/test/electron-browser/api/mock.ts"
+		// "./vs/base/browser/browser.ts",
```

Comment:
Do we need to disable all the VS Code ones? Shouldn't they pass correctly unless we've made changes that don't follow strict null, in which case we should update those changes?

Reply:
We don't have the work they have done to make all their stuff pass; so including theirs adds a lot of noisy errors. Maybe once we get the next merge most of their errors should be fixed